.github/workflows/gradle-build-pr.yml

```diff
@@ -36,13 +36,11 @@ jobs:
           java-version: 17
           distribution: 'zulu'
       - uses: gradle/gradle-build-action@v2
-        env:
-          CI: true
```

Comment:
The `CI` variable is added by default, see https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables